CHANGELOG.md

```diff
@@ -56,6 +56,10 @@ host and/or host with a different port
   - Updated `getS3KeyForArchivedMessage` in `ingest/sqs` to store SQS messages
     by `queueName`.
 
+### Changed
```

Comment:
This should be merged with the "Changed" section above

Reply:
You had me freaked out for a second, because I was sure I checked that.  Thanks "merge branch 'master'". https://github.com/nasa/cumulus/commit/5877d8caa60809b901dbd31f4e1b5883ec0a8271
I'll fix it